src/_posts/2016-08-26-Continuous-Integration-on-iOS-with-HockeyApp-and-Testflight-Deployment.html

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
Did you convert this to html? We should still use markdown for posts. This is one of the strengths of Jekyll and it simplifies the writing process since people can just write in markdown.

Reply:
Yes I did, in order to be able to use Bootstrap in posts but now I see that we only need grids in pages where we see lists so actual content can be in markdown and also Jekyll injects it into div in html anyway. So I have just moved old posts.
However, what about the images in posts. Is there a custom functionally that gets images from them then copy those into **_post_images** folder?

Reply:
Yes, Jekyll takes care of converting markdown into html. For images we usually upload them somewhere else (e.g. on NStack) and then link them directly in the post or they get added to this repo. There's no custom functionality for Jekyll to do this. I think we should align the process and document it in the README. Feel free to consider this and come up with a suggestion.

Reply:
I will try to find a way at last. For example, there can be a custom plugin gets images from the posts. You may also create a **ToDo** for this on Trello.